app/assets/javascripts/full_calendar.js

```diff
@@ -1,21 +1,9 @@
-window.onload = function () {
-    if (document.getElementById('calendar') != null) {
-       calendar()
+$(function () {
+    document.getElementById('calendar') != null
+    {
+        calendar()
     }
```

Comment:
Really? What about if-else

Reply:
It bad to use if-else. What will you do if you need to load 10 different js scripts?

Reply:
your window onload breaks other scripts. Use jquery
http://learn.jquery.com/using-jquery-core/document-ready/